src/requirements/requirement-frontend-computation.ts

```diff
@@ -12,22 +13,13 @@ type FulfillmentStatistics = {
   readonly courses: readonly (readonly CourseTaken[])[];
 } & RequirementFulfillmentStatistics;
 
-/**
- * @param course course object with useful information retrived from Cornell courses API.
- * @returns true if the course is AP/IB equivalent course or credit
- */
-const courseIsAPIB = (course: CourseTaken): boolean =>
-  [CREDITS_COURSE_ID, FWS_COURSE_ID].includes(course.courseId) ||
-  ['AP', 'IB'].includes(course.code.split(' ')[0]);
-
 /**
  * Used for total academic credit requirements for all colleges except EN and AR
  * @param course course object with useful information retrived from Cornell courses API.
  * @returns true if the course is not PE or 10** level
  */
 const courseIsAllEligible = (course: CourseTaken): boolean => {
-  if (course.courseId === CREDITS_COURSE_ID) return true;
-  if (courseIsAPIB(course)) return false;
+  if (courseIsAPIB(course)) return true;
```

Comment:
This `courseIsAPIB(course)` check is still necessary to ensure that AP/IB transfer credits are always counted (even if, for some reason, they conflict with `subject !== 'PE' && !number.startsWith('10')`).

The previous line is no longer necessary, because it's included in the `courseIsAPIB` function.